src/DynamoSandbox/Program.cs

```diff
@@ -22,6 +22,8 @@ internal class PathResolver : IPathResolver
         private readonly List<string> additionalResolutionPaths;
         private readonly List<string> additionalNodeDirectories;
         private readonly List<string> preloadedLibraryPaths;
+        private readonly string userDataRootFolder;
+        private readonly string commonDataRootFolder;
```

Comment:
These members are not needed, simply return `string.Empty` from the `get` would do.